massgov/mayflower-patternlab
# DP-8924: reorder org page to have featured topics above news

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description
This PR reorders the organization page to have featured topics above news

## Related Issue / Ticket

- [JIRA issue](https://jira.state.ma.us/browse/DP-8924)


## Steps to Test


1. View the organization page
2. Ensure the order of the sections is correct as per this document: https://docs.google.com/document/d/1JnC_IPJEjyqxqtICGPoITkzTxdfjK_tgmuJzNCKE5tE/edit


#### Impacted Areas in Application


* organization page JSON
